airflow/models.py

```diff
@@ -3828,6 +3848,29 @@ def get_dag(self):
         return self.dag
 
     @provide_session
+    def get_previous_dagrun(self, session=None):
```

Comment:
please rename to "get_previous", it is already part of DagRun (or even make it a property)